CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Immobilienscout Changelog
 
-## v0.0.3
-
-Update the response accord to the Immobilienscout changes in the `JSON` structure
-
-## v0.0.3
+## v0.0.2
```

Comment:
Are there a few points missing here? For now it should not be commited the new version number, instead the first header should be something like "unreleased" until we really release and tag the new version. This should be a separate commit on master.